README.md

```diff
@@ -1 +1,152 @@
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">Visual Studio</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Github pages</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>No database was  used</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">NA</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Mobile version with Mobile menu]**
+- **[A Navbar with a banna]**
+- **[Speakers section dynamically generated by JS]**
+- **[Desktop Version]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://enocol.github.io/capstone/about.html)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+- create a local directory on your computer.
+- Open a Terminal and run the following code
+
+git clone https://github.come/enocol/capstone.git
+
+Note: You must have git running on your computer.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+### Install
+
+Install this project with:
+
+### Usage
+
+To run the project, execute the following command:
+open the project in your refered code Editors and run a local live server.
+
+### Run tests
+
+To run tests, run the following command:
+
+### Deployment
+
+You can deploy this project using:
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Enoh Collins**
+
+- GitHub: [@githubhandle](https://github.com/enocol)
+- Twitter: [@twitterhandle](https://twitter.com/enocol)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/enocol)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] Kindly remove this comment from your `README` file because the comment is for developer understanding related to how to create a professional `README` file for the project. Having this kind of comment makes your `README` file looks unprofessional.